src/status_im2/contexts/syncing/syncing_instructions/view.cljs

```diff
@@ -0,0 +1,122 @@
+(ns status-im2.contexts.syncing.syncing-instructions.view
```

Comment:
i feel like it's not worth it to have such a complex component, simpler will be to have just two separate views in same namespace, which will be reusing some parts